client/dashboard/app/queries/user-tax-details.ts

```diff
@@ -0,0 +1,20 @@
+import { queryOptions, mutationOptions } from '@tanstack/react-query';
```

Comment:
Can we rename the filename prefix from `user-` to `me-`, for consistency with existing files?

Reply:
Renamed in e5e42b2.